base/shell/cmd/lang/tr-TR.rc

```diff
@@ -1,386 +1,384 @@
-/* TRANSLATOR: 2015 Erdem Ersoy (eersoy93) (erdemersoy [at] erdemersoy [dot] net) */
+/* TRANSLATOR: 
+2015 Erdem Ersoy (eersoy93) (erdemersoy [at] erdemersoy [dot] net),
+2021 Süleyman Poyraz (zaryob [dot] dev [at] gmail [dot] com)
+*/
 
-LANGUAGE LANG_TURKISH, SUBLANG_DEFAULT
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```

Comment:
:eyes:

Reply:
Oww you find why its complation failed. Im gonna change it brother. Thanx.

Reply:
I first overwrote en-US.rc files over tr-Tr.rc and re-translated to see the translation missing of all files. For this reason, it may have overlooked. For a few hours I've been compiling the ROS and trying to fix the problem.

Reply:
It seems its ok for now.